docs/usecases/update-primary-component-metadata/README.md

```diff
@@ -0,0 +1,37 @@
+# Updating Primary Component Metadata
+
+## Description
+
+A common usecase is for a software producer to create an SBOM using a tool like trivy, syft, or other tool in a CI/CD pipeline. The generated SBOM often lacks the metadata for license, copyright, descriptions, and external references like documentation.
+
+## Desire Input
+
+- [An SBOM of any format](metadata.cyclonedx.json) that only contains the metadata that needs to be updated and associated
+identifier to update.
+- A generated SBOM of any format that comes from syft, trivy, or other tool
+
+## Desired Output
+
+- SBOMs in SPDX and CDX that have the generated SBOM and updated metadata
```

Comment:
I wonder if a use case here would be creating an "empty" SBOM to be filled out during a review process or something